### DISCUSSION TITLE
Dialog not closing

### DISCUSSION THREAD

**Discussion by Teellcory** [vuetifyjs/vuetify#12469] [Q&A] [answered]
To preface: I can't make a codepen example of the scenario because I can't reproduce the scenario outside of my environment, as I don't know what aspect(s) is causing the issue. 

I have a dialog (A) that has buttons to open two other dialogs (B & C). Dialog A remains open when either B or C is open. B & C can't be open together. B behaves completely as expected. C is almost a verbatim copy of B, except with its own model and so forth. I have tried clearing out everything except for a card that simply has a title and the problem still occurs.

I have a close button that is supposed to set the model to false. It does, yet the dialog doesn't close. Post-clicking the button, it will not close until I have caused another event to happen, such as toggling a radio button. Watching the events, these are the results:

Good dialog
- click - VBtn
- update:return-value - VDialog

Bad dialog
- click - VBtn
- click - VCard

I'm looking for advice on what could cause that event chain, as that seems to be the only clue to the problem.

**Comment by Renato66** [comment]
can you record it? 

https://shareit.video

**Comment by Teellcory** [comment, marked as answer]
I rebuilt it from scratch and it worked this time.
